biblib/views/user_view.py

```diff
@@ -58,9 +58,24 @@ def retrieve_user_email(owner_absolute_uid):
                     service
                 )
         return response
+    
+    @classmethod
+    def get_user_libraries(cls, session, service_uid, sort_col, sort_order, type):
```

Comment:
I think this method looks good and also makes that previous query a lot more readable.